ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/e2e/Pagination/home-page-pagination.cy.js

```diff
@@ -2,25 +2,28 @@
 
 describe('101092: Pagination', () => {
 
-	beforeEach(() => {
-		cy.login()
-	})
+    beforeEach(() => {
+        cy.login()
+    })
 
     after(function () {
         cy.clearLocalStorage()
     })
 
     it('TC01: Next/Previous button if project number is greater than 10', () => {
         cy.get('[test-id="projectCount"]')
-        .invoke('text')
-        .then((text) => {
-            if (text.includes('Projects (10)')) {
-                cy.get('[test-id="nextPage"]').click()
-                cy.get('[test-id="previousPage"]').click()
-            }
-            else {
-                cy.log('Number of projects does not exceed 10')
-            }
-        })
+            .invoke('text')
+            .then((text) => {
+                cy.log(text)
+                const total_count = text.replace('projects found', '').trim()
+                if (total_count > 10) {
+                    cy.get('[test-id="nextPage"]').click()
+                    cy.url().should('contain', 'currentPage=2');
+                    cy.get('[test-id="previousPage"]').click()
```

Comment:
This is to check if user has clicked on Next then user can get the `Previous` link and click it 
I can add a assertion to check page number if that make more sense :)

Reply:
I'm showing my unfamiliarity with Cypress 😄

Sounds like line 22 would fail the test if the element with the specified ID was missing, or it was not possible to click it.

An extra assertion on the page number would make it a bit clearer, though.